pandas/tests/groupby/test_groupby.py

```diff
@@ -233,6 +233,39 @@ def test_apply_issues(self):
             lambda x: x['time'][x['value'].idxmax()])
         assert_series_equal(result, expected)
 
+    def test_apply_trivial(self):
+        # GH 20066
+        def gen_one_df(start=0):
```

Comment:
you dont need a function for this, just create the dataframe directly

Reply:
done in my last commit.